test/html.test.js

```diff
@@ -665,3 +672,88 @@ QUnit.test('does not wrap mp4 source buffers', function() {
   );
   QUnit.equal(mediaSource.sourceBuffers.length, 2, 'created native buffers');
 });
+
+QUnit.test('can get activeSourceBuffers', () => {
```

Comment:
Good point. Will change. And you're doubly right because tests are better suited to not using fat arrows, as we are using "this" a lot in them.